File: Codechef/Practice/Parking Charges.cpp
https://www.codechef.com/problems/PARKINGCHARG




#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int x,y,h;
	cin >> x >> y >> h;
	cout << x + (h-1)*y;
	return 0;
}

File: Codechef/Practice/Parking Charges.go
https://www.codechef.com/practice/course/go/PGOL02/problems/PARKINGCHARG




package main
import "fmt"

func main(){
	// your code goes here
	var x,y,h int
	fmt.Scan(&x,&y,&h)
	
	total:= x + y*(h-1)
	
	fmt.Println(total)
}